File: transferreports-ui/transferreports/src/app/models/transfer-rumor.ts
export type Credibility = 'Tier1' | 'Tier2' | 'Gossip';

export interface TransferRumor {
  id: string;
  player: string;
  fromClub: string;
  toClub: string;
  fee?: string;
  source: string;
  sourceUrl?: string;
  credibility: Credibility;     
  timestamp: string;        
  note?: string;
}

File: transferreports-api/TransferReports.Api/Models/TransferRumor.cs
namespace TransferReports.Api.Models;

public class TransferRumor
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string Player { get; set; } = "";
    public string FromClub { get; set; } = "";
    public string ToClub { get; set; } = "";
    public string? Fee { get; set; }           // "€65m", "Free", "Loan"
    public string Source { get; set; } = "";   // "Sky Sports"
    public string? SourceUrl { get; set; }
    public Credibility Credibility { get; set; } = Credibility.Tier2;
    public DateTimeOffset Timestamp { get; set; } = DateTimeOffset.UtcNow;
    public string? Note { get; set; }          // short quote
}
